e2e/tests/functional/forms.e2e.spec.js

```diff
@@ -128,6 +128,75 @@ test.describe('Persistence operations @couchdb', () => {
             timeout: 1000
         }).toEqual(1);
     });
+    test('Can create an object after a conflict error @couchdb', async ({ page }) => {
```

Comment:
Let's add the 2p annotation for anything which instantiates a second browser. We'll need to track this use case as we start to execute these tests with multiple browser types and targets. (Most likely we'll need to skip them)

Reply:
Does it still count if it only opens a second tab within the same browser context? or anything with multiple pages should be marked 2p?

Reply:
i think let's use it for both pages and contexts